Bottom: 5fa9e42fdbdc0a4f503cf8bb04c50955e914e1c7
Top:    a40e625037bf0a6e49f312e334f8193501053535
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-27 15:44:50 +0100

Refresh of hid-gtk-encapsulate-view-param

---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index 8672b19..0c9019d 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -134,7 +134,7 @@ ghid_zoom_view_fit (void)
 }
 
 static void
-gport_flip_view (Coord center_x, Coord center_y, bool flip_x, bool flip_y)
+ghid_flip_view (Coord center_x, Coord center_y, bool flip_x, bool flip_y)
 {
   int widget_x, widget_y;
 
@@ -1387,15 +1387,15 @@ SwapSides (int argc, char **argv, Coord x, Coord y)
       switch (argv[0][0]) {
         case 'h':
         case 'H':
-          gport_flip_view (gport->pcb_x, gport->pcb_y, true, false);
+          ghid_flip_view (gport->pcb_x, gport->pcb_y, true, false);
           break;
         case 'v':
         case 'V':
-          gport_flip_view (gport->pcb_x, gport->pcb_y, false, true);
+          ghid_flip_view (gport->pcb_x, gport->pcb_y, false, true);
           break;
         case 'r':
         case 'R':
-          gport_flip_view (gport->pcb_x, gport->pcb_y, true, true);
+          ghid_flip_view (gport->pcb_x, gport->pcb_y, true, true);
           Settings.ShowSolderSide = !Settings.ShowSolderSide; /* Swapped back below */
           break;
         default:
